guides/common/modules/con_limiting-resources.adoc

```diff
@@ -0,0 +1,18 @@
+[id="limiting-resources"]
+= Limiting resources
+
+You can use the Resource Quota plugin to limit users' access to resources.
+
+Supported resource types::
+* CPU {endash} limit the number of CPU cores
+* Memory {endash} limit the amount of memory
+* Disk {endash} limit the amount of disk space
```

Comment:
```suggestion
* Disk space {endash} limit the amount of disk space
```
To be consistent with the web UI label below.

Reply:
Applied; also fixed "CPU" which is called "CPU cores" in WebUI.